File: SecondSemester/OOP/Lab5/C#/Circle.cs
using System;

namespace Lab5
{
    public class Circle : Figure
    {
        private float Radius { get; set; }

        public Circle(float radius)
        {
            Radius = radius;
        }

        public override double GetArea()
        {
            return Math.PI * Math.Pow(Radius, 2);
        }

        public override double GetPerimeter()
        {
            return 2 * Math.PI * Radius;
        }
    }
}

File: SecondSemester/OOP/Lab5/C++/Circle.cpp
//
// Created by bohdan on 07.04.20.
//

#include "Circle.h"
#include "math.h"



Circle::Circle(float radius)
{
    Radius = radius;
}

double Circle::GetArea()
{
    return M_PI* pow(Radius, 2);
}

double Circle::GetPerimeter()
{
    return 2 * M_PI * Radius;
}

